packages/website/src/components/RulesTable/index.tsx

```diff
@@ -0,0 +1,178 @@
+          mode === 'exclude' && styles.checkboxLabelDeactivated,
+        )}
+        onKeyDown={(e): void => {
+          if (e.key === 'Enter') {
+            toNextMode();
+          }
+        }}
+        onClick={toNextMode}
+        aria-label={`Toggle the filter mode. Current: ${mode}`}
+      >
+        {label}
+      </button>
+    </li>
+  );
+}
+
+function match(mode: FilterMode, value: boolean): boolean | undefined {
+  if (mode === 'exclude') {
+    return !value;
+  }
+  if (mode === 'include') {
+    return value;
+  }
+  return undefined;
+}
+
+export default function RulesTable({
+  extensionRules,
+}: {
+  extensionRules?: boolean;
+}): JSX.Element {
+  const rules = useRulesMeta();
+  const [showRecommended, setShowRecommended] = useState<FilterMode>('neutral');
+  const [showStrict, setShowStrict] = useState<FilterMode>('neutral');
+  const [showFixable, setShowFixable] = useState<FilterMode>('neutral');
+  const [showHasSuggestions, setShowHasSuggestion] =
+    useState<FilterMode>('neutral');
+  const [showTypeCheck, setShowTypeCheck] = useState<FilterMode>('neutral');
+  const relevantRules = useMemo(
+    () =>
+      rules
+        .filter(r => !!extensionRules === !!r.docs?.extendsBaseRule)
+        .filter(r => {
+          const opinions = [
+            match(
+              showRecommended,
+              r.docs?.recommended === 'error' || r.docs?.recommended === 'warn',
+            ),
+            match(showStrict, r.docs?.recommended === 'strict'),
+            match(showFixable, !!r.fixable),
+            match(showHasSuggestions, !!r.hasSuggestions),
+            match(showTypeCheck, !!r.docs?.requiresTypeChecking),
+          ].filter((o): o is boolean => o !== undefined);
+          return opinions.every(o => o);
+        }),
+    [
+      rules,
+      extensionRules,
+      showRecommended,
+      showStrict,
+      showFixable,
+      showHasSuggestions,
+      showTypeCheck,
+    ],
+  );
+  return (
+    <>
+      <ul className={clsx('clean-list', styles.checkboxList)}>
+        <RuleFilterCheckBox
+          mode={showRecommended}
+          setMode={setShowRecommended}
+          label="✅ recommended"
+        />
+        <RuleFilterCheckBox
+          mode={showStrict}
+          setMode={setShowStrict}
+          label="🔒 strict"
+        />
+        <RuleFilterCheckBox
+          mode={showFixable}
+          setMode={setShowFixable}
+          label="🔧 fixable"
+        />
+        <RuleFilterCheckBox
+          mode={showHasSuggestions}
+          setMode={setShowHasSuggestion}
+          label="🛠 has suggestions"
+        />
+        <RuleFilterCheckBox
+          mode={showTypeCheck}
+          setMode={setShowTypeCheck}
+          label="💭 requires type information"
+        />
+      </ul>
+      <table>
+        <thead>
+          <tr>
+            <th>Rule</th>
+            <th>✅🔒</th>
+            <th>🔧🛠</th>
+            <th>💭</th>
```

Comment:
these were some widths that worked well on windows 10 chrome and provide a bit of a buffer around the emoji
![image](https://user-images.githubusercontent.com/7462525/171347109-d859f150-2263-4702-9281-2698fc107d82.png)

***NOTES***:
- you need to use `width:100%` on the first column to force the browser to expand it to fill remaining space
- you need to use `min-width`! The browser will ignore `width` because table layout rules area fun. `min-width` sets a lower-bound constraint which the browser will prioritise over the `width: 100%`

```suggestion
            <th style={{width: '100%'}}>Rule</th>
            <th style={{minWidth: 70}}>✅🔒</th>
            <th style={{minWidth: 70}}>🔧🛠</th>
            <th style={{minWidth: 50}}>💭</th>
```